fixtures/tmb/solaris-flare.json

```diff
@@ -0,0 +1,850 @@
+          "name": "Burst (use rate at full)"
+        },
+        {
+          "range": [246, 250],
+          "name": "Meltdown: Random Pixels with solid background"
+        },
+        {
+          "range": [251, 255],
+          "name": "Meltdown: Random pixels with burst background"
+        }
+      ]
+    },
+    "Red Strobe": {
+      "type": "Single Color",
+      "color": "Red"
+    },
+    "Green Strobe": {
+      "type": "Single Color",
+      "color": "Green"
+    },
+    "Blue Strobe": {
+      "type": "Single Color",
+      "color": "Blue"
+    },
+    "White Strobe": {
+      "type": "Single Color",
+      "color": "White"
+    }
+  },
+  "templateChannels": {
+    "$pixelKey Red": {
+      "type": "Single Color",
+      "color": "Red"
+    },
+    "$pixelKey Green": {
+      "type": "Single Color",
+      "color": "Green"
+    },
+    "$pixelKey Blue": {
+      "type": "Single Color",
+      "color": "Blue"
+    },
+    "$pixelKey White": {
+      "type": "Single Color",
+      "color": "White"
+    }
+  },
+  "modes": [
+    {
+      "name": "Basic RGB 8bit",
+      "shortName": "RGB8bit",
+      "rdmPersonalityIndex": 1,
+      "channels": [
+        "Red",
+        "Green",
+        "Blue"
+      ]
+    },
+    {
+      "name": "Basic RGB 16bit",
+      "shortName": "RGB16bit",
+      "rdmPersonalityIndex": 2,
+      "channels": [
+        "Red",
+        "Red fine",
+        "Green",
+        "Green fine",
+        "Blue",
+        "Blue fine"
+      ]
+    },
+    {
+      "name": "Basic RGBW 8bit",
+      "shortName": "RGBW8bit",
+      "rdmPersonalityIndex": 3,
+      "channels": [
+        "Red",
+        "Green",
+        "Blue",
+        "White"
+      ]
+    },
+    {
+      "name": "Basic RGBW 16bit",
+      "shortName": "RGBW16bit",
+      "rdmPersonalityIndex": 4,
+      "channels": [
+        "Red",
+        "Red fine",
+        "Green",
+        "Green fine",
+        "Blue",
+        "Blue fine",
+        "White",
+        "White fine"
+      ]
+    },
+    {
+      "name": "Strobe Only",
+      "shortName": "strobe",
+      "rdmPersonalityIndex": 5,
+      "channels": [
+        "Flash intensity",
+        "Flash duration",
+        "Flash rate",
+        "Flash effects"
+      ]
+    },
+    {
+      "name": "RGB Strobe",
+      "shortName": "RGBstrobe",
+      "rdmPersonalityIndex": 6,
+      "channels": [
+        "Red Strobe",
+        "Green Strobe",
+        "Blue Strobe",
+        "Flash intensity",
+        "Flash duration",
+        "Flash rate",
+        "Flash effects"
+      ]
+    },
+    {
+      "name": "RGBW Strobe",
+      "shortName": "RGBWstrobe",
+      "rdmPersonalityIndex": 7,
+      "channels": [
+        "Red Strobe",
+        "Green Strobe",
+        "Blue Strobe",
+        "White Strobe",
+        "Flash intensity",
+        "Flash duration",
+        "Flash rate",
+        "Flash effects"
+      ]
+    },
+    {
+      "name": "Advanced RGB Strobe 1 Pixel",
```

Comment:
Please also use hyphens in mode names, to make it clear that it's 1 pixel, not strobe (mode) 1.